recipe/meta.yaml

```diff
@@ -44,6 +45,7 @@ test:
     - cellxgene_census
   requires:
     - pip
+    - python ={{ python_min }}
```

Comment:
This runs the tests with the minimum supported Python version, which I really like since it will catch any potential problems early